### ISSUE TITLE
Edit: "Adding a new feature" page

### ISSUE THREAD

**Issue by jknipes** [department-of-veterans-affairs/va.gov-team#86407]
## Issue Description

- Adding a New Feature page requires an update regarding language on Drupal vs. `vagov-content`.
We have determined the IIR team can help update the language on that page.
[Most recent Slack thread re: next steps](https://dsva.slack.com/archives/C05RJS5DANT/p1718396060050739)
---
## Tasks
- [x] Work with SME to draft changes to the page
- [x] Update page and prepare for deployment

## SME:

- Nick Sayre
## Related ticket: 
[74464](https://github.com/department-of-veterans-affairs/va.gov-team/issues/74464)

## Acceptance Criteria
- [ ] Page is updated and ready for deployment
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by jknipes** [comment]
cc: @nicks

**Comment by taylojill** [comment]
I updated two pages: Adding applications and features page and Creating new content. See summary below. I sent pdf copy of changes to Nick Sayre (SME) via platform content channel.
-  Added a blue info box stating the vagov-content repo is still being used by several teams
-  I added a link to the GitHub Readme file provided by SME. 
-  I reworded one sentence on one page to clarify that Drupal is now the primary source of content (blue color font).

Waiting for feedback.
